tests/servers/tests.py

```diff
@@ -27,6 +29,34 @@
 }
 
 
+class LiveServerThreadTest(TestCase):
+
+    def run_live_server_thread(self, connections_override=None):
+        thread = LiveServerTestCase._create_server_thread(connections_override)
+
+        thread.daemon = True
+        thread.start()
+        thread.is_ready.wait()
+        thread.terminate()
+        thread.join()
+
+    @unittest.skipIf(connection.vendor == 'sqlite', "sqlite does not permit closing connections")
```

Comment:
Btw, if you're curious, the special-casing for SQLite goes back to 2006: https://github.com/django/django/commit/6f87b17a0d0da75767d0bbf05d52f787e897c599